assets/src/components/unsaved-changes-warning/index.js

```diff
@@ -20,9 +20,10 @@ import { Options } from '../options-context-provider';
  *
  * @param {Object} props Component props.
  * @param {boolean} props.excludeUserContext Whether to exclude listening to user context.
+ * @param {Node} props.appRoot React app root.
```

Comment:
Technically this should be `Element` not `Node`, right?

```suggestion
 * @param {Element} props.appRoot React app root.
```